lxd/network/driver_ovn.go

```diff
@@ -3104,6 +3161,13 @@ func (n *ovn) Start() error {
 		return err
 	}
 
+	// If VLAN config is provided to the OVN network, configure the necessary ports
+	// to handle traffic on the specified VLAN.
+	err = n.configurePortsForVLAN(n.config["vlan"], false)
```

Comment:
Can we make this called, or be part of, startUplinkPort() as that can then avoid loading the uplink network from the DB again, and also it keeps the uplink port configuration together, rather than splitting it out.